spanner/transaction.go

```diff
@@ -165,16 +165,20 @@ type ReadOptions struct {
 
 	// The request tag to use for this request.
 	RequestTag string
+
+	// If this is for a partitioned read and DataBoostEnabled field is set to true, the request will be executed via Spanner independent compute resources.
```

Comment:
nit: split this comment over two lines

Also add the following to the end of the comment: `Setting this option for regular read operations has no effect.`

Reply:
Added Knut.